### ISSUE TITLE
No output file generated in SchemaCrawler

### ISSUE THREAD

**Issue by TamilJackson** [daniel-werner/laravel-schemacrawler#10]
No output files generated in SchemaCrawler

While using below command, getting no error in cmd
php artisan schema:generate
Generated diagram to D:\xampp\htdocs\public_html\configurationTool\storage\app\schema.pdf

But in storage\app folder, no files are there.

Using /schema route in application showing below error since it has no files
_Symfony\Component\HttpFoundation\File\Exception\FileNotFoundException
The file "D:\xampp\htdocs\public_html\configurationTool\storage\app\schema.png" does not exist_

Having error in cmd output file too
C:\Users\lenovo\AppData\Local\Temp\sf_proc_00.err
_Error: dot: can't open D:\xampp\htdocs\public_html\configurationTool\storage\app\schema.png: No such file or directory_

**Comment by daniel-werner** [comment]
Thanks for reporting the issue Tamil.

As this package used Graphviz to generate the diagram, do you have it installed on your machine? https://graphviz.org/download/

I did not test this on windows, please let me know if installing Graphviz helped, thanks.

**Comment by TamilJackson** [comment]
Yes. I have already installed Graphviz for windows. After installing Graphviz, dot cmd was recognized, but not generating output file. Output folder permission is also already given.

**Comment by quyle92** [comment]
The same problem here. I've successfully install Graphviz for Window10 (can assure that as I also use "beyondcode/laravel-er-diagram-generator" package which requires  Graphviz  as dependency) but no output was generated.
